src/dialects/abstract/query-interface.d.ts

```diff
@@ -274,6 +274,25 @@ export interface IndexDescription {
   type: string | undefined;
 }
 
+export interface ConstraintDescription {
+  constrainCatalog: string | null | undefined;
```

Comment:
```suggestion
  constraintCatalog?: string;
```
Both a typo and a suggestion. If we really want it to allow `null` as well you can add that but I'd prefer not to. The suggestion holds for the other properties in this interface as well

Reply:
I've updated the typing now, let me know if you're happy with this.